Objects/object.c

```diff
@@ -14,7 +14,7 @@ Py_ssize_t _Py_RefTotal;
 Py_ssize_t
 _Py_GetRefTotal(void)
 {
-    PyObject *o;
+    PyObject *o;cmp
```

Comment:
This `cmp` doesn't look it will compile?

Reply:
uh, github edit, let me verify, locally.

Reply:
I have done the needful